backend/src/main/java/de/neuefische/cgnjava222/productgallery/model/SingleOrderDetails.java

```diff
@@ -0,0 +1,10 @@
+package de.neuefische.cgnjava222.productgallery.model;
+
+import java.util.List;
+
+public record SingleOrderDetails(
+        String id,
+        String userName,
+        List<OrderDetailsItem> orderItems
```

Comment:
Ist es nicht möglich 2 Liste: mit OrderItems und mot OrderDetailItem zusammen (in 2 Felder) speichern und anstatt SingleOrder und SingleOrderDetails eine Model haben?